.github/workflows/branch.yml

```diff
@@ -1,6 +1,6 @@
 name: nf-core branch protection
 # This workflow is triggered on PRs to main branch on the repository
-# It fails when someone tries to make a PR against the phac-nml `main` branch instead of `dev`
```

Comment:
For this particular `branch.yml` file, could you change all the mentions of `nf-core` in this file to `phac-nml` (like it was before copying the newer `branch.yml` over)? Since it's' then clearer that this isn't an nf-core maintained pipeline.

Since modifications to `branch.yml` will trigger an issue with linting, you can then add a line to `.nf-core.yml` to ignore checking this file (like https://github.com/phac-nml/iridanextexample/blob/dev/.nf-core.yml#L21)

Reply:
Of course!
Fixed in [bfdd0c8](https://github.com/phac-nml/speciesabundance/pull/4/commits/bfdd0c853b6ddd2772b47cd4406077cc56782695)
:)

Reply:
Looks great. Thanks so much 😄